doc/users/next_whats_new/url_in_images_for_pdfs.rst

```diff
@@ -0,0 +1,10 @@
+URL Support for Images in PDF Backend
+-------------------------------------
+
+The PDF backend can now generate clickable images if a URL is provided to the
+image. There are a few limitations worth noting though:
+
+* If parts of the image are clipped, the non-visible parts are still clickable.
+* If there are transforms applied to the image, the whole enclosing rectangle
+  is clickable. However, if you use ``interpolation='none'`` for the image,
+  only the transformed image area is clickable (depending on viewer support).
```

Comment:
Sorry, I'm confused about what this means.

Reply:
First sentence: Basically, the image with the hand drawn "rectangle" above is clickable. Probably there is a better wording for it.

Second sentence: if you rely on the PDF transform (i.e. set interpolation='none' and possibly a few more conditions) the clickable area is the image. Bracket part: not all PDF viewers support that and will fall back on the red rectangle.